pymongo/topology.py

```diff
@@ -430,21 +477,21 @@ def _get_replica_set_members(self, selector):
             ):
                 return set()
 
-            return {sd.address for sd in selector(self._new_selection())}
+            return {sd.address for sd in iter(selector(self._new_selection()))}
 
-    def get_secondaries(self):
+    def get_secondaries(self) -> Set[_Address]:
         """Return set of secondary addresses."""
         return self._get_replica_set_members(secondary_server_selector)
 
-    def get_arbiters(self):
+    def get_arbiters(self) -> Set[_Address]:
         """Return set of arbiter addresses."""
         return self._get_replica_set_members(arbiter_server_selector)
 
-    def max_cluster_time(self):
+    def max_cluster_time(self) -> Optional[Mapping[str, Any]]:
```

Comment:
Let's use ClusterTime here and anywhere else we use cluster_time or $clusterTime.

Reply:
yes! done!